### ISSUE TITLE
How can I get emsg messageData using compiled script?

### ISSUE THREAD

**Issue by miyukki** [google/shaka-player#717]
- What version of Shaka Player are you using?
  v2.0.6

  - Can you reproduce the issue with our latest release version?
    Yes, use dist/shaka-player.compiled.js.
  - Can you reproduce the issue with the latest code from `master`?
    No, use original source.

- Are you using the demo app or your own custom app?
  my custon app

  - If custom app, can you reproduce the issue using our demo app?
    I'm not tried, maybe demo apps does not contain emsg box.

- What browser and OS are you using?
  electron@v1.6.1

- What did you do?
  I want to get emsg data via MP4 segments.
  But I can't get the messageData because `shaka.dash.DashParser.EmsgInfo` was compiled.

- What did you expect to happen?
```javascript
player.addEventListener('emsg', function(event) {
  // event.detail.messageData represents emsg data
});
```

- What actually happened?
```javascript
player.addEventListener('emsg', function(event) {
  // event.detail.pd represents emsg data
});
```

How can I get emsg messageData using compiled script? Or, do you have a best way to get?

**Comment by TheModMaker** [comment]
This has been fixed in the `master` branch, so it will appear in v2.1.  I have also pushed a change to the v2.0.x branch.  You can pull that branch and build yourself if you want to use v2.0.x. It will appear in the next bugfix release (if we do one).

**Comment by joeyparrish** [comment]
We will do a v2.0.7 release in the near future, with this and any other accumulated bug fixes.  Thanks!

**Comment by joeyparrish** [comment]
This was just released in v2.0.7.
